.github/workflows/coverage.yml

```
@@ -49,6 +49,7 @@ jobs:
         source ./.venv/bin/activate
         python -m pip install -U coverage
         python -m test.pythoninfo
+        export PYTHONPATH=./.venv/lib/python3.9/site-packages/coverage/fullcoverage
```

This is how the `python3.9` is built looking at https://github.com/python/cpython/blob/master/Lib/venv/__init__.py#L129-L131:

```suggestion
        export PYTHONPATH=./.venv/lib/$(./python -c "import sys; print('python%d.%d' % sys.version_info[:2])")/site-packages/coverage/fullcoverage
```

---

That is a clever idea! Thinking about it, `export PYTHONPATH=./.venv/lib/$(ls .venv/lib | grep python | head -n 1)/site-packages/coverage/fullcoverage` would also work and might be a bit simpler as it doesn't depend on `sys`, it just gets the name of that python* directory.

---

Whatever works and is accepted by a core dev is fine by me, you can also use `find .venv -name fullcoverage` which may be more robust.

---

yeah I like that one the best, I've updated it now